templates/js/ai-assistant-bot/README.md.tpl

```diff
@@ -0,0 +1,99 @@
+# Overview of the AI Assistant Bot template
+
+This app template is built on top of [Teams AI library](https://aka.ms/teams-ai-library) and [OpenAI Assistants API](https://platform.openai.com/docs/assistants/overview).
+It showcases how to build an intelligent chat bot in Teams capable of helping users accomplish a specific task using natural language right in the Teams conversations, such as solving a math problem.
+
+- [Overview of the AI Assistant Bot template](#overview-of-the-ai-assistant-bot-template)
+  - [Get started with the AI Assistant Bot template](#get-started-with-the-ai-assistant-bot-template)
+  - [What's included in the template](#whats-included-in-the-template)
+  - [Extend the AI Assistant Bot template with more AI capabilities](#extend-the-ai-assistant-bot-template-with-more-ai-capabilities)
+  - [Additional information and references](#additional-information-and-references)
+
+## Get started with the AI Assistant Bot template
+
+> **Prerequisites**
+>
+> To run the AI Assistant Bot template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-cli)
+> - An account with [OpenAI](https://platform.openai.com/).
+>
+> **Note**
+>
+> Teams AI Library is currently in preview version.
+
+### Create your own OpenAI Assistant
+
+Before running or debugging your bot, please follow these steps to setup your own OpenAI Assistant.
+
+**If you haven't setup any Assistant yet**
```

Comment:
I suggest saying something about create.ts to show we provide the script to create assistant, customize if needed.

Reply:
Added:

> This app template provides script `src/creator.ts` to help create assistant. You can customize the instructions and settings there.